service/src/main/java/bio/terra/workspace/service/workspace/model/EnumeratedJobs.java

```diff
@@ -0,0 +1,36 @@
+package bio.terra.workspace.service.workspace.model;
+
```

Comment:
this class feels like a good candidate for java autovalue. https://www.baeldung.com/introduction-to-autovalue

Reply:
The current guidance has a preference for using `immutable.io` over autovalue. I have a personal preference for being able to see the code, especially while debugging, rather than have it magicked up, so I never use them. If you want to use them, feel free, but please use immutable. It has much better and cleaner support for Jackson serdes.